packages/xo-web/src/xo-app/pool/tab-patches.js

```diff
@@ -166,8 +170,32 @@ const INSTALLED_PATCH_COLUMNS = [
 }))
 @connectStore({
   hostPatches: createGetObjectsOfType('patch').pick((_, { master }) => master.patches),
+  runningVms: createGetObjectsOfType('VM').filter(
+    createSelector(
+      (_, props) => props.pool.id,
+      poolId => vm => vm.$pool === poolId && vm.power_state === 'Running'
```

Comment:
Don’t we need to consider `suspended` and `paused` VMs?

Reply:
Issue only said "show it disabled if some powered up VMs are using a non-shared storage", I think suspended and paused VMs are not considered running, are they?

Reply:
I think we only need to check running VMs because we only evacuate running VMs during the RPU: https://github.com/vatesfr/xen-orchestra/blob/b0c37df8d73beb9589dc28e32d5a4409a48a9272/packages/xo-server/src/xapi/mixins/patching.mjs#L539-L543